### DISCUSSION TITLE
Upload only one specific file

### DISCUSSION THREAD

**Discussion by vinsol007** [abraunegg/onedrive#1749] [Q&A] [answered]
Is it possible to upload just a specific file instead of an entire folder?

Congrats for this really nice app!

**Comment by abraunegg** [comment, marked as answer]
@vinsol007 
> Is it possible to upload just a specific file instead of an entire folder?

This is not currently possible - the closest you can get is to use `--single-directory` to specify where that file exists.

**Comment by vinsol007** [comment]
Thanks for your answer @abraunegg
